web_src/js/features/repo-projects.js

```diff
@@ -141,6 +141,21 @@ export function initRepoProject() {
     window.location.reload();
   });
 
+  $(document).on('click', '.unset-default-project-board', async function (e) {
+    e.preventDefault();
+
+    await $.ajax({
+      method: 'POST',
+      url: $(this).data('url'),
```

Comment:
@silverwind use `attr` instead of `data`?

Reply:
Didn't check this in detail as it just copied existing code above. But I agree we should refactor all these cases to `this.getAttribute('data-url')`.

Reply:
Need to be fixed in this PR?

Reply:
Would say we can do in another.